### ISSUE TITLE
Add option to deactivate uninstallation of packages

### ISSUE THREAD

**Issue by FlorentJeannot** [jazzband/pip-tools#1444]
#### What's the problem this feature will solve?

Being able to keep installed packages in the venv which are not specified in the `requirements.txt`.

For example, it could be that I want `memory-profiler` installed in my venv but I don't want to specify it as a dependency for the project.

#### Describe the solution you'd like

I'd like to pass an option to `pip-sync` so that it does not uninstall my packages.

For example to disable it completely:

```
pip-sync --disable-uninstall
```

#### Alternative Solutions

I don't see any good alternative, but here's a list of suggestions:
1. Do `pip-sync` first, install packages in venv after (but next `pip-sync` wille remove them)
2. Use `pip install -r requirements.txt` (I think I have read somewhere that `pip` should not be used directly with requirements generated with `pip-tools`, but I am not sure)

Does this feature make sense to you?
If it does, I am willing to provide a PR. Also if you find a better name than `--disable-uninstall`, please feel free to share.

Thanks for your feedback!

**Comment by atugushev** [comment]
Discussed here: https://github.com/jazzband/pip-tools/issues/686.

TLDR; `pip install -r requirements.txt` is the solution.

**Comment by FlorentJeannot** [comment]
> Discussed here: #686.
> 
> TLDR; `pip install -r requirements.txt` is the solution.

Thanks for your feedback, I'm sorry I didn't see it was already discussed.
I think we can close this issue then?

**Comment by atugushev** [comment]
No problem 👍
